kokoro/build_binaries.sh

```diff
@@ -14,16 +14,16 @@ export RUST_BACKTRACE=1
 export RUST_LOG=debug
 export XDG_RUNTIME_DIR=/var/run
 
-readonly SCRIPTS_DIR="$(dirname "$0")"
+readonly SCRIPTS_DIR="$(dirname "$0" | cut -d/ -f1)"
 # shellcheck source=scripts/common
-source "$SCRIPTS_DIR/common"
+source "$SCRIPTS_DIR/scripts/common"
```

Comment:
This seems wrong: why is the SCRIPTS_DIR followed by `scripts`?

Reply:
Thanks. Fixed.